From: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Subject: kstrtox: remove strtobool()
Date: Sat, 16 Sep 2023 17:50:11 +0200

The conversion from strtobool() to kstrtobool() is completed.  So
strtobool() can now be removed.

Link: https://lkml.kernel.org/r/87e3cc2547df174cd5af1fadbf866be4ef9e8e45.1694878151.git.christophe.jaillet@wanadoo.fr
Signed-off-by: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Reviewed-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/kstrtox.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/include/linux/kstrtox.h~kstrtox-remove-strtobool
+++ a/include/linux/kstrtox.h
@@ -147,9 +147,4 @@ extern long simple_strtol(const char *,c
 extern unsigned long long simple_strtoull(const char *,char **,unsigned int);
 extern long long simple_strtoll(const char *,char **,unsigned int);
 
-static inline int strtobool(const char *s, bool *res)
-{
-	return kstrtobool(s, res);
-}
-
 #endif	/* _LINUX_KSTRTOX_H */
_
